src/dispatching/dispatcher.rs

```diff
@@ -83,6 +87,17 @@ where
         Self { worker_queue_size: size, ..self }
     }
 
+    /// Maximum number of inactive workers.
+    ///
+    /// When number of workers exceeds this limit dispatcher will try to remove
+    /// inactive workers.
+    ///
+    /// By default it's 32.
+    #[must_use]
+    pub fn gc_worker_count_trigger(self, count: usize) -> Self {
```

Comment:
How does a user know a value for this configuration parameter? When it'd be needed to alter the default of 32?

Reply:
This parameter changes how much memory teloxide is spending. The more users and memory you have, the higher this parameter can be and vice-versa. But I'm not sure how to choose a value for it.

Reply:
I guess it should be a little higher than the average (?) number of users (well, things with different distribution keys) so that "GC" is not triggered for nothing.

Reply:
But then again, the higher this parameter is/the more users there are, the longer "GC" pause will be, uhhhhhh. I wish we knew a better way to do all of this.